### PR TITLE
travis: run e2e tests with ci-aeon or with a special commit message

### PR DESCRIPTION
**Pull request by broady**
FORCE_E2E=true in the commit message will trigger e2e tests even if not
triggered nightly by ci-aeon.

(Yes, it's put in this message intentionally!)

### CODE REVIEW

**Comment by adg** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +30,21 @@ before_install:
...
+      export GOLANG_SAMPLES_E2E_TEST=true;
+    fi;
+  fi
+- if [ $GOLANG_SAMPLES_E2E_TEST ]; then
+    echo "This test run will run end-to-end tests.";
+    go get -u google.golang.org/appengine/cmd/aedeploy;
+    export PATH="$PATH:$PWD/google-cloud-sdk/bin";
+    ./testing/travis/configure_gcloud.bash;
+  fi
+- env
```
Comment (tmatsuo) [inline .travis.yml]: Also print out `gcloud info` if you haven't done other place (it will help if you want to file an issue).

Comment (tmatsuo) [inline .travis.yml]: maybe in `configure_gcloud.bash`?

**Comment by tmatsuo** [comment]
neat idea :) LGTM

File: .travis.yml (hunk 1 above)
Comment (broady) [inline .travis.yml]: Good idea. Added.

**Pull request merged**